res/css/structures/_ScrollPanel.scss

```diff
@@ -21,6 +21,6 @@ limitations under the License.
         display: flex;
         flex-direction: column;
         justify-content: flex-end;
-        overflow-y: hidden;
```

Comment:
Why is this change needed...? When testing the PR deploy preview here, I didn't notice any difference if I reverted this line back to just `overflow-y`...

Reply:
If you switch to the IRC layout a horizontal scroll bar appears. I originally did that as a temporary fix and accidentally committed it :sweat_smile: There is probably a better solution - I'll look into it

![Screenshot_20210415_175057](https://user-images.githubusercontent.com/25768714/114899306-24914700-9e13-11eb-9bc1-dcf3cc8d8b71.png)

Reply:
Hmmm, with the line removed everything seems to work fine. Hopefully, it makes sense